src/lexedata/change/clean_ids.py

```diff
@@ -53,6 +53,47 @@ def clean_mapping(rows: t.Mapping[str, t.Mapping[str, str]]) -> t.Mapping[str, s
     return mapping
 
 
+def update_integer_ids(ds: pycldf.Dataset, table: csvw.metadata.Table):
```

Comment:
@Anaphory I think `mapping: t.Mapping[str, str]` might be missing as a function argument here